src/content/app/genome-browser/components/track-panel/components/track-panel-list/TrackPanelList.tsx

```diff
@@ -113,6 +116,9 @@ export const TrackPanelList = () => {
                   <AccordionItemButton
                     className={accordionButtonClassNames}
                     disabled={!category.track_list.length}
+                    onClick={() =>
+                      trackTrackPanelSectionToggled(category.label)
+                    }
```

Comment:
I am not sure I am seeing this event.

Reply:
```
Processing GTAG command: ["event", "section_toggled", {event_action: "section_toggled", event_category: "track_panel", event_label: "Genes & transcripts", species: "Human - GRCh38.p13", app: "Genome browser", feature: "gene - ENSG00000139618"}]
```

Is Lucy interested whether the section has expanded/collapsed, or is she just interested that someone clicked on it?